exporter/chronicleexporter/README.md

```diff
@@ -47,6 +47,9 @@ currently supported log types are:
 - windows_event.system
 - sql_server
 
+
+If the `attributes["chronicle_log_type"]` field is present in the log, we will use it's value in the payload instead of the automatic detection or the `log_type` in the config.
```

Comment:
```suggestion
If the `attributes["chronicle_log_type"]` field is present in the log, we will use its value in the payload instead of the automatic detection or the `log_type` in the config.
```